tests/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.10
+
+ADD ./runner /
+ADD ./tracee-trainer/* /
+RUN chmod +x /runner
```

Comment:
probably these should be last layers

Reply:
https://github.com/aquasecurity/tracee/pull/1282/commits/6d3be669a2e7ce253508aa49560260168816f4bf